src/airport.js

```diff
@@ -1,5 +1,34 @@
+const Plane = require('./Plane.js') //only used to check instance of - is there a way around this?
+
 class Airport {
-  // here's a starting point for you
+  planesAtAirport = []
+
+  constructor(name, airportCapacity = 10) {
+    if (typeof (airportCapacity === 'number') && airportCapacity > 0) {
+      this.airportCapacity = airportCapacity
+    } else { this.airportCapacity = "Capacity not set, must be a positive integer" };
+    this.name = name;
+  };
+
+  atCapacity = () => {
+    return this.planesAtAirport.length === this.airportCapacity ? true : false
```

Comment:
I would check if I need the '? true: false' or would it automatically return true and false anyway?